builders/build/local/adding-built-in-pallet.md

```diff
@@ -13,7 +13,16 @@ What sets Substrate apart is its modular architecture, which enables the seamles
 
 For cases requiring only EVM (Ethereum Virtual Machine) compatibility, the template provided in the [Tanssi repository](https://github.com/moondance-labs/tanssi#container-chain-templates){target=_blank} fulfills the requirements without further modifications. However, teams aiming to build a Substrate Appchain must add and configure both built-in and custom modules within the runtime. This involves compiling, generating the chain specification, and deploying through the Tanssi protocol to transform it into a live ContainerChain.
 
-This article focuses on the necessary steps for adding a built-in module.
+This article focuses on the necessary steps for adding a built-in module to the EVM template.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To follow the steps in this guide, you will need to have the following:
+
+- A healthy development environment with the Rust compiler and Cargo package manager
+- The [Tanssi repository](https://github.com/moondance-labs/tanssi){target=_blank}, cloned from Github
```

Comment:
More than this, they need to have the Tanssi EVM Container Template compiled right?